test-cases/longevity/longevity-1TB-5days-authorization-and-tls-ssl.yaml

```diff
@@ -6,11 +6,8 @@ prepare_write_cmd: ["cassandra-stress write cl=QUORUM n=275050075 -schema 'repli
 
 stress_cmd: ["cassandra-stress mixed cl=QUORUM duration=6800m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=100 -pop seq=1..550100150  -log interval=5 -col 'size=FIXED(200) n=FIXED(5)'",
              "cassandra-stress mixed cl=QUORUM duration=6800m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=100 -pop seq=550100151..1100200300  -log interval=5 -col 'size=FIXED(200) n=FIXED(5)'",
-             "cassandra-stress user profile=/tmp/cs_mv_profile.yaml ops'(insert=3,read1=1,read2=1,read3=1)' cl=QUORUM duration=6800m -mode cql3 native -rate threads=10"]
-
-
-stress_read_cmd: ["cassandra-stress read cl=QUORUM duration=6800m -mode cql3 native  -rate threads=10 -pop seq=1..1100200300  -log interval=5 -col 'size=FIXED(200) n=FIXED(5)'" ]
-
+             "cassandra-stress user profile=/tmp/cs_mv_profile.yaml ops'(insert=3,read1=1,read2=1,read3=1)' cl=QUORUM duration=6800m -mode cql3 native -rate threads=10",
```

Comment:
It is not only `read`, it is `mixed` because it has `insert=3` part.

Reply:
@vponomaryov , your comment is not related... this line is presented as changed because a `,` was added to the end, so the next line moved from `stress_read_cmd` to `stress_cmd` list of commands